src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/FilteredTokens.java

```diff
@@ -27,32 +27,35 @@ public FilteredTokens(ArrayList<Token> tokens) {
         this.tokens = tokens;
     }
 
+
+
     public FilteredTokens(int idToken, String filteredTokens){
         this.idToken = idToken;
         this.filteredTokens = filteredTokens;
     }
 
-    public int getIdToken() {
+    int getIdToken() {
         return idToken;
     }
 
-    public String getFilteredTokens() {
+    String getFilteredTokens() {
         return filteredTokens;
     }
 
-    private ArrayList<String> tokenToString(ArrayList<Token> tokens) {
+    static ArrayList<String> tokenToString(ArrayList<Token> tokens) {
```

Comment:
Meant to be default and not private modifier?

Reply:
done